bots/elasticsearch/bot.py

```diff
@@ -79,67 +79,67 @@ class ElasticSearchBot(Bot):
         'creativeworks': {
             'dynamic': False,
             'properties': {
-                'affiliations': {'type': 'string', 'fields': EXACT_FIELD},
-                'contributors': {'type': 'string', 'fields': EXACT_FIELD},
+                'affiliations': {'type': 'text', 'fields': EXACT_FIELD},
+                'contributors': {'type': 'text', 'fields': EXACT_FIELD},
                 'date': {'type': 'date', 'format': 'strict_date_optional_time', 'include_in_all': False},
                 'date_created': {'type': 'date', 'format': 'strict_date_optional_time', 'include_in_all': False},
                 'date_modified': {'type': 'date', 'format': 'strict_date_optional_time', 'include_in_all': False},
                 'date_published': {'type': 'date', 'format': 'strict_date_optional_time', 'include_in_all': False},
                 'date_updated': {'type': 'date', 'format': 'strict_date_optional_time', 'include_in_all': False},
-                'description': {'type': 'string'},
-                'funders': {'type': 'string', 'fields': EXACT_FIELD},
-                'hosts': {'type': 'string', 'fields': EXACT_FIELD},
-                'id': {'type': 'string', 'index': 'not_analyzed', 'include_in_all': False},
-                'identifiers': {'type': 'string', 'fields': EXACT_FIELD},
-                'justification': {'type': 'string', 'include_in_all': False},
-                'language': {'type': 'string', 'index': 'not_analyzed', 'include_in_all': False},
-                'publishers': {'type': 'string', 'fields': EXACT_FIELD},
-                'registration_type': {'type': 'string', 'index': 'not_analyzed', 'include_in_all': False},
+                'description': {'type': 'text'},
+                'funders': {'type': 'text', 'fields': EXACT_FIELD},
+                'hosts': {'type': 'text', 'fields': EXACT_FIELD},
+                'id': {'type': 'text', 'index': 'not_analyzed', 'include_in_all': False},
```

Comment:
I think fetching a document by ID doesn't actually use this field (`hits[0]._id` vs `hits[0]._source.id`), but I agree we should leave it searchable.